PR.jenkinsfile

```diff
@@ -0,0 +1,36 @@
+// Norma CI test norma using "make test"
+
+pipeline {
+	agent { label 'norma' }
+	
+	options {
+		timestamps ()
+		timeout(time: 3, unit: 'HOURS')
+	}
+
+	stages {
+		stage('Check Norma Format') {
+			steps {
+				catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
```

Comment:
Can you remind me what does `buildResult 'SUCCESS' mean`?

Reply:
The intention is for the pipeline to proceed to build and run tests despite failing gofmt check.
This should be marked as `buildResult: 'UNSTABLE'` instead.